dissect/target/helpers/docker.py

```diff
@@ -0,0 +1,136 @@
+import re
+from typing import Union
+
+from dissect.cstruct import cstruct
+
+from dissect.target.helpers.protobuf import ProtobufVarint
+
+# Resources:
+# - https://github.com/moby/moby/pull/37092
+# - https://github.com/cpuguy83/docker/blob/master/daemon/logger/local/doc.go
+# - https://github.com/moby/moby/blob/master/api/types/plugins/logdriver/entry.proto
+local_def = """
+struct entry {
+    uint32   header;
+
+    // source
+    uint8    s_type;        // 0x0a
+    varint   s_len;         // 0x06
+    char     source[s_len]; // stdout or stderr
+
+    // timestamp
+    uint8    t_type;        // 0x10
+    varint   ts;            // timestamp in ums
+
+    // message
+    uint8    m_type;        // 0x1a
+    varint   m_len;         // message length
+    char     message[m_len];
+
+    // partial_log_metadata not implemented
+
+    uint32 footer;
+};
+"""
+
+c_local = cstruct(endian=">")
+c_local.addtype("varint", ProtobufVarint(cstruct=c_local, name="varint", size=1, signed=False, alignment=1))
+c_local.load(local_def, compiled=False)
+
+DOCKER_NS_REGEX = re.compile(r"\.(?P<nanoseconds>\d{7,})(?P<postfix>Z|\+\d{2}:\d{2})")
+
+
+def convert_timestamp(timestamp: str) -> str:
+    """
+    Docker sometimes uses (unpadded) 9 digit nanosecond precision
```

Comment:
```suggestion
    """Docker sometimes uses (unpadded) 9 digit nanosecond precision
```
We also have a 120 character or so line limit, don't feel constrained to use it ;) also applies to the other docstrings.